### PR TITLE
Fix log_parser.py. It broke after we renamed ActionDictionary to DedupInfo.

### PR DESCRIPTION
**Pull request by kumpera**
I should probably add some tests to it in conjunction to example_gen.

### CODE REVIEW

**Review by olgavrou — APPROVED** [review]

**Review by cheng-tan — COMMENTED** [review]
Line3 needs to import `DedupInfo`

File: test_tools/log_parser/parser.py [hunk 1]
```diff
@@ -38,7 +39,9 @@ def process_payload(payload, is_dedup):
 def parse_cb(payload):
     evt = cb.CbEvent.GetRootAsCbEvent(payload, 0)
     print(f'\tcb: actions:{evt.ActionIdsLength()} ctx:{evt.ContextLength()} probs:{evt.ProbabilitiesLength()} lm:{learning_mode_name(evt.LearningMode())}')
-    print('\tcontext: ' + bytearray(evt.ContextAsNumpy()).decode('utf-8'))
+    payload = json.loads(bytearray(evt.ContextAsNumpy()).decode('utf-8'))
+    print('\tcontext:')
+    print(json.dumps(payload, indent = 1))
 
 def parse_action_dict(payload):
     evt = ad.ActionDictionary.GetRootAsActionDictionary(payload, 0)
```
Comment (cheng-tan) [inline test_tools/log_parser/parser.py:47]: Here needs to be updated to DedupInfo

File: test_tools/log_parser/parser.py [hunk 2]
```diff
@@ -5,8 +5,9 @@
 import reinforcement_learning.messages.flatbuff.v2.PayloadType as pt
 import reinforcement_learning.messages.flatbuff.v2.LearningModeType as lm
 import flatbuffers
-import zstd
+import zstandard as zstd
```
Comment (cheng-tan) [inline test_tools/log_parser/parser.py:8]: Line 37, zstd.decompress(payload) needs to be updated to:
```
dctx = zstd.ZstdDecompressor()
return dctx.decompress(payload)
```

Comment (kumpera) [reply test_tools/log_parser/parser.py:8]: You have the wrong pip package installed, the above line works with zstandard

**Comment by kumpera** [comment]
@cheng-tan the PR now should deal with all issues you mentioned

**Pull request merged**